### DISCUSSION TITLE
2024-13-24-housekeeping-update

### DISCUSSION THREAD

**Discussion by giscus** [haroohie-club/HaroohieSite#112] [Announcements]
# 2024-13-24-housekeeping-update

Jonko debriefs the status of the translation patches as well as provides updates on new website features and upcoming blog posts.

https://haroohie.club/blog/2024-13-24-housekeeping-update

**Comment by Cepher777** [comment]
niiiice keep up the good work, I like how you guys casually reverse engineer games like its nothing. Its only a short matter of time before you have Hieretsu completely figured out I bet.
